content/resources/kubecon/index.md

```diff
@@ -26,26 +26,26 @@ block_external_search_index: true
 
 # The url slug for the webinar landing page. If this is an external
 # webinar, use the external URL as the value here.
-url_slug: https://pulumi.com/kubecon/
+url_slug: /kubecon
 
 # Content for the left hand side section of the page.
 main:
     # Webinar title.
-    title: KubeCon + CloudNativeCon Europe 2024
+    title: KubeCon North America
 
     event_type: event # workshop | event
 
     # URL for embedding a URL for ungated webinars.
     youtube_url:
 
     # Sortable date. The datetime Hugo will use to sort the webinars in date order.
-    sortable_date: 2024-03-19T09:00:00-00:00
+    sortable_date: 2024-11-12T09:00:00-07:00
 
     # Duration of the webinar.
     duration: 1 hour
 
     # "virtual" will be shown under "show virtual events only", otherwise shown as City, State (seattle, wa)
-    location: Paris, France
+    location: Salt Lake City
```

Comment:
```suggestion
    location: Salt Lake City, UT
```

not sure if this even matters, but just going by what it says in the comment above. if not just ignore.